src/shipit_pipeline/shipit_pipeline/api.py

```diff
@@ -7,13 +7,21 @@
 import logging
 
 from shipit_pipeline.pipeline import get_runnable_steps, get_running_steps, refresh_pipeline_steps, start_steps,\
-    PipelineStep
-
+    PipelineStep, MockPipelineStep
+mock1 = MockPipelineStep("mock1", {"timeout": 2}, [], "pending", 2)
+mock2 = MockPipelineStep("mock2", {"timeout": 5}, ["mock1"], "pending", 5)
+mock3 = MockPipelineStep("mock3", {"timeout": 10}, ["mock2"], "pending", 10)
+mock4 = PipelineStep("mock4", "https://tset",{"timeout": 2}, [], "pending")
 log = logging.getLogger(__name__)
 
-PIPELINES = {
-
-}
+PIPELINES = { "pipelinetest":["running",{
+              "parameters": {
+                "steps": [mock1]
+              },
+              "uid": "pipelinetest"
+            }]
+            }
+MockSteps = {"mock1":mock1, "mock2":mock2, "mock3":mock3}
```

Comment:
We don't need this variable. everything should be pulled from PIPELINES_SPECS and PIPELINES_STATE